00 Meta/Vault Infos/Introduce Use Cases.md

```diff
@@ -0,0 +1,44 @@
+# Introduce Use Cases
```

Comment:
`# Use Cases` makes more sense as header to me.

Reply:
yeah, you're right 🤔 started the file differently and then didn't adjust the title :) Thanks!